src/games/GameShow.tsx

```diff
@@ -35,13 +73,24 @@ const GameBoardView = () => {
   return <GameGrid victoryState={victoryState} boardState={boardState} />;
 };
 
+const GameId = () => {
+  const id = useFieldValue({ source: "id" });
+  return <div className="game-infos">Game - id: {id}</div>;
+};
+
 export const GameShow = () => (
   <Show>
     <SimpleShowLayout>
-      <MatchupField />
+      <GameId></GameId>
+      <span className="RaLabeled-label">Players: </span>
```

Comment:
no need to change (this is just FYI): you could probably have used a [Labeled](https://marmelab.com/react-admin/Labeled.html) here